DependencyInjection/CmfCoreExtension.php

```diff
@@ -118,9 +118,6 @@ public function prepend(ContainerBuilder $container)
                         break;
                     case 'cmf_routing':
                         $routePaths = array($persistenceConfig['basepath'].'/routes');
```

Comment:
maybe just make sure the routing bundle knows the base path for its default behaviour?